### PR TITLE
Display app even for unknown networks

### LINKED ISSUE #324: why are you limiting the app to specific networks?
is there a technical reason why it shouldnt run on any deployment of gnosis safe on any network?
why does it have to support only "official" ones?

### PR DESCRIPTION
**Pull request by bh2smith**
If this works, this PR closes #324

By removing the "unsupported network" page for unknown networks. 

This version is deployed at IPFS hash: QmTef26dDY2XABj84ioRBpFk7vZPRmSkNJ2eYJeKMnTMk9

and can be tested by loading this app into the safe interface 

https://bafybeiepy5xhantx776xnp5pb3442de3kp4owmoc36e5ni23optcauam44.ipfs.infura-ipfs.io/

Please try it out @sirpy and let us know if this is what you were hoping for.

### CODE REVIEW

**Comment by sirpy** [comment]
@bh2smith 
need to remove unused vars
is this going to be deployed to ipfs once it passes build?

**Comment by schmanu** [comment]
Hey @bh2smith ,

I think we should at least put a warning that the app has not officially been released / tested for the chosen network.

Otherwise this looks good.

File: src/App.tsx [hunk 1]
```diff
@@ -51,7 +50,7 @@ const App: React.FC = () => {
   return (
     <Container>
       <Header />
-      {networkMap.has(safe.chainId) ? (
```
Comment (schmanu) [inline src/App.tsx:54]: Here we should add a warning or a small warning banner on top of the app. 
We could also separate this into an issue and I add it in January.

**Review by schmanu — APPROVED** [review]
👍 this should work and I feel like the all in general should work on arbitrary networks.

File: src/App.tsx (hunk 1 above)
Comment (bh2smith) [reply src/App.tsx:54]: I would prefer to keep that separate (not even sure what the warning should say) and let you do it in January if you don't mind.

Comment (bh2smith) [reply src/App.tsx:54]: Issue created here https://github.com/bh2smith/safe-airdrop/issues/326

**Comment by sirpy** [comment]
@bh2smith perfect... it worked. thanks!

**Comment by bh2smith** [comment]
Cool then we will merge this but I won't plan to publish a new release until a few more fixes come in. Alright for you to use the deployed version linked here?

**Pull request merged**